.gitignore

```diff
@@ -2,3 +2,6 @@ vendor/
 .openapi-generator
 var/
 .env
+
+### Jetbrains IDE's ###
+.idea/
```

Comment:
I’m personally against such changes, I prefer to keep gitignore tech stack agnostic. Today it's intelliji, tomorrow vs code, later something even more exotic.
My recommendation is to use global gitignore for such things but since I’m not going to maintain this repo, will let you decide 🙂

Reply:
I understand this and will revert the addition.